src/parser/ad_parser.js

```diff
@@ -265,10 +289,10 @@ function parseWrapper(wrapperElement, emit) {
  * @param  {Node[]} extensions - The array of extensions to parse.
  * @return {AdExtension[]} - The nodes parsed to extensions
  */
-export function _parseExtensions(extensions) {
+export function _parseExtensions(extensions, type = 'Ad') {
```

Comment:
please update jsdocs :)

Reply:
also as a bonus, you could also move this function to a new file.

Reply:
Done for both